docs/operational_guide/availability_consistency_durability.md

```diff
@@ -0,0 +1,126 @@
+# Tuning Availability, Consistency, and Durability
+
+## Overview
+
+M3DB is designed as a High Availability [HA](https://en.wikipedia.org/wiki/High_availability) system because it doesn't use a consensus protocol like Raft or Paxos to enforce strong consensus and consistency guarantees.
+However, even within the category of HA systems, there is a broad spectrum of consistency and durability guarantees that a database can provide.
+To address as many use cases as possible, M3DB can be tuned to achieve the desired balance between performance, availability, durability, and consistency.
+
+Generally speaking, [the default and example configuration for M3DB](https://github.com/m3db/m3/tree/master/src/dbnode/config) favors performance and availability, as that is well-suited for M3DB's most common metrics and Observability use cases. To instead favor consistency and durability, consider tuning values as described in (TODO).
```

Comment:
i think i'd suggested copy with Observability capitalized, but let's generalize this a bit more, e.g.:

> [...] that is well-suited for M3DB's most common use cases (such as metric storage for observability). [...]